charactersheet/charactersheet/models/common/weapon.js

```diff
@@ -136,6 +136,27 @@ function Weapon() {
         }
     });
 
+    self.weaponRangeLabel = ko.pureComputed(function() {
+        if (self.weaponType().toLowerCase() === 'ranged'){
```

Comment:
Little nit pick here, is there are reason to check of ranged __and__ melee? Can we change this to just check on or the other. All you'd need to is:
```
if (weaponType === 'ranged') {
    ...
} else {
    ...
}
```